build.gradle

```diff
@@ -678,8 +678,8 @@ uiTestTargets.each { version ->
 
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
-      environment 'IDEPROBE_SCREENSHOTS_DIR', System.getProperty("user.home") + '/artifacts/uiTest' + version + '/screenshots'
-      environment 'IDEPROBE_BASE_PATH', System.getProperty("user.home")
+      environment 'IDEPROBE_PATHS_SCREENSHOTS', System.getProperty("user.home") + '/artifacts/uiTest' + version + '/screenshots'
+      environment 'IDEPROBE_PATHS_BASE', System.getProperty("user.home")
```

Comment:
Whoops in case of local runs (no CI, no VM, no Docker)... won't that pollute home directory with a few new directories (of little importance, other than for UI testing)?

Reply:
TBH I'd wrap both `environment 'IDEPROBE_PATHS_...', ...` statements into e.g. `if (!isCi) { ... }`... or sth like that 🤔

Reply:
so maybe wrap `IDEPROBE_PATHS_BASE` into `if (!isCI) { ... }.` ? but  in that case I need to write in CONTRIBUTING.md that this variable must be set when starting the container

Reply:
Okay, let's do it this way

Reply:
Maybe actually... use sth like `$HOME/.ideprobe/` for both base path and screenshots? I mean, using non-dotted (`.*`) directories under home directory like `~/artifacts` is still pretty unconventional/surprising :/